.travis.yml

```diff
@@ -7,61 +7,17 @@ jobs:
     # Linux
     - name: "aarch64-unknown-linux-gnu"
       env: TARGET=aarch64-unknown-linux-gnu NO_JEMALLOC_TESTS=1
-    - name: "arm-unknown-linux-gnueabi"
-      env: TARGET=arm-unknown-linux-gnueabi NO_JEMALLOC_TESTS=1
-    - name: "armv7-unknown-linux-gnueabihf"
-      env: TARGET=armv7-unknown-linux-gnueabihf NO_JEMALLOC_TESTS=1
-    - name: "i586-unknown-linux-gnu"
-      env: TARGET=i586-unknown-linux-gnu
-      addons: &gcc_multilib
-        apt:
-          packages:
-            - gcc-multilib
-    - name: "i686-unknown-linux-gnu (nightly)"
-      env: TARGET=i686-unknown-linux-gnu
-      addons: *gcc_multilib
-    - name: "i686-unknown-linux-gnu (beta)"
-      env: TARGET=i686-unknown-linux-gnu
-      addons: *gcc_multilib
-      rust: beta
-    - name: "i686-unknown-linux-gnu (stable)"
-      env: TARGET=i686-unknown-linux-gnu
-      addons: *gcc_multilib
-      rust: stable
-    # FIXME: blocked onhttps://github.com/jemalloc/jemalloc/issues/1464
-    # - name: "i686-unknown-linux-musl"
-    #   env: TARGET=i686-unknown-linux-musl NOBGT=1 NO_JEMALLOC_TESTS=1
-    - name: "mips-unknown-linux-gnu"
-      env: TARGET=mips-unknown-linux-gnu NO_JEMALLOC_TESTS=1
-    - name: "mips64-unknown-linux-gnuabi64"
-      env: TARGET=mips64-unknown-linux-gnuabi64 NO_JEMALLOC_TESTS=1
-    - name: "mips64el-unknown-linux-gnuabi64"
-      env: TARGET=mips64el-unknown-linux-gnuabi64 NO_JEMALLOC_TESTS=1
-    - name: "mipsel-unknown-linux-gnu"
-      env: TARGET=mipsel-unknown-linux-gnu NO_JEMALLOC_TESTS=1
-    - name: "powerpc-unknown-linux-gnu"
-      env: TARGET=powerpc-unknown-linux-gnu NO_JEMALLOC_TESTS=1
-    - name: "powerpc64-unknown-linux-gnu"
-      env: TARGET=powerpc64-unknown-linux-gnu NO_JEMALLOC_TESTS=1
+      arch: arm64
     - name: "powerpc64le-unknown-linux-gnu"
       env: TARGET=powerpc64le-unknown-linux-gnu NO_JEMALLOC_TESTS=1
+      arch: ppc64le
```

Comment:
Are we going to support the PowerPC arch?

Reply:
Actually we had run the tests for several months.